tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -162,6 +163,10 @@ def bookmark(self, bookmark_title):
     def number_of_columns(self):
         return int(self.marionette.find_element(*self._landing_page_locator).get_attribute('cols'))
 
+    @property
+    def is_homescreen_visible(self):
+        return self.marionette.find_element(*self._homescreen_container_locator)
```

Comment:
`find_element` will return an [HTMLElement](http://marionette-client.readthedocs.org/en/latest/reference.html#marionette.HTMLElement) which is an non-empty object so Python would [always consider it to be true here](https://docs.python.org/2.7/library/stdtypes.html?highlight=truth#truth-value-testing). So, this test can give some false positive results :cry: 

We can use [displayed](http://marionette-client.readthedocs.org/en/latest/reference.html#marionette.HTMLElement.is_displayed) to make sure the homescreen is here.